### DISCUSSION TITLE
Request: Integration with https://web3alert.io/

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#16] [Integration]
This will be useful for on-chain gov, on-chain event notification etc.

@furoxr Would you follow up on this?

**Comment by hackfisher** [comment]
Alternative: https://www.notifi.network/

**Reply by hackfisher** [reply]
https://github.com/kianenigma/polkadot-basic-notification/issues

**Reply by wuminzhe** [reply]
https://github.com/open-web3-stack/guardian

**Comment by furoxr** [comment]
I'll take a look~
